proto-public/pbtenancy/v1alpha1/namespace.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package hashicorp.consul.tenancy.v1alpha1;
+
+// The name of the Namespace is in the outer Resource.ID.Name.
+// It must be unique within a partition and must be a
+// DNS hostname. There are also other reserved names that may not be used.
+message Namespace {
+  // Description is where the user puts any information they want
+  // about the namespace. It is not used internally.
+  string description = 1;
+  // ACLs is the configuration of ACLs for this namespace. It has its
+  // own struct so that we can add more to it in the future.
+  // This is nullable so that we can omit if empty when encoding in JSON
+  ACLConfig acls = 2;
+}
+
+// ACLConfig is the Namespace specific ACL configuration container.
+// This is a copy of NamespaceACLConfig from v1 protos.
+message ACLConfig {
+  // PolicyDefaults is the list of policies that should be used for
+  // the parent authorizer of all tokens in the associated namespace.
+  repeated ACLLink policy_defaults = 1;
+
+  // RoleDefaults is the list of roles that should be used for the
+  // parent authorizer of all tokens in the associated namespace.
+  repeated ACLLink role_defaults = 2;
+}
+
+// This is a copy of acl.ACLLink from v1 protos.
+message ACLLink {
```

Comment:
I'm not sure about adding this as part of the namespace proto. Maybe we need to move them when we get to adding partitions.

Reply:
I'm fine with removing ACLConfig and ACLLink until we can address them in later phases.

Reply:
yeah I think it's better, as so far we don't really need them.